src/mir/passes.md

```diff
@@ -53,6 +53,18 @@ want to use a [MIR visitor]. MIR visitors are a handy way to walk all
 the parts of the MIR, either to search for something or to make small
 edits.
 
+### MIR optimization levels
+
+MIR optimizations can come in various levels of readiness. Experimental
+optimizations may cause miscompilations, or slow down compile times.
+However, these passes are complex enough to be included in nightly builds
+even in these states, to enable validating them or otherwise gather feedback.
```

Comment:
I don't think it's because they're _complex_, it's so they're in-tree and multiple people can work on them at once.
```suggestion
These passes are still included in nightly builds to gather feedback and make it easier to modify the pass.
```

Reply:
I don't think I wanted to use complex there but I can't remember. Thanks for the suggestions!